include/TGUI/Widgets/TextArea.hpp

```diff
@@ -219,6 +219,30 @@ TGUI_MODULE_EXPORT namespace tgui
         TGUI_NODISCARD std::size_t getMaximumCharacters() const;
 
 
+        /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+        /// @brief Changes the string that is inserted when the Tab key is pressed
+        ///
+        /// @param tabtext  The string to insert when Tab is pressed.
+        ///
+        /// Defaults to "\t".
+        ///
+        /// @warning Any cartridge return characters given in this string (\r) will be removed!
+        ///
```

Comment:
Perhaps you can also add a warning here that the tab key doesn't affect the text area unless `gui.setTabKeyUsageEnabled(false)` was called.